tests/functional/database_change/test_varchar_widening.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import os
+from dbt.tests.util import run_dbt
+from dbt.tests.tables import TableComparison
+
+
+incremental_sql = """
+{{
+  config(
+    materialized = "incremental"
+  )
+}}
+
+select * from {{ this.schema }}.seed
+
+{% if is_incremental() %}
+
+    where id > (select max(id) from {{this}})
+
+{% endif %}
+"""
+
+materialized_sql = """
+{{
+  config(
+    materialized = "table"
+  )
+}}
+
+select * from {{ this.schema }}.seed
+"""
+
+
+@pytest.fixture
+def models():
+    return {"incremental.sql": incremental_sql, "materialized.sql": materialized_sql}
+
+
+def test_varchar_widening(project):
+    path = os.path.join(project.test_data_dir, "varchar10_seed.sql")
+    project.run_sql_file(path)
```

Comment:
This would work but I think we are moving everything to dbt seed, one example is [link](https://github.com/dbt-labs/dbt-core/blob/main/tests/functional/graph_selection/fixtures.py#L173)

Reply:
This check needs to make changes directly to the database.  It's a slightly different use case.

Reply:
Yeah, for this case the point is that the definition of the table in the database changes. At the very least the second sql file would have to be run, so it's probably more straightforward to have them both be sql.

Reply:
[seed.sql](https://github.com/dbt-labs/dbt-core/blob/0.21.latest/test/integration/007_graph_selection_tests/seed.sql#L10) this is the SQL file seed replaced, why it is a different use case?